services/docker-compose.yml

```diff
@@ -300,6 +300,10 @@ services:
       R_CLONE_OPTION_TRANSFERS: ${R_CLONE_OPTION_TRANSFERS}
       R_CLONE_PROVIDER: ${R_CLONE_PROVIDER}
 
+      EFS_DNS_NAME: ${EFS_DNS_NAME}
+      EFS_MOUNTED_PATH: ${EFS_MOUNTED_PATH}
+      EFS_ENABLED_FOR_USERS: ${EFS_ENABLED_FOR_USERS}
```

Comment:
I would suggest to use the JSON version since it's gonna be easier `DYNAMIC_SIDECAR_EFS_SETTINGS` you have only 1 env var.

Reply:
Yes that is also valid approach, I will leave it like this for now.